### PR TITLE
Move action buttons to have enough space to chat

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

**Review by herflis — APPROVED** [review]

File: apps/sensenet/src/components/document-viewer.tsx [hunk 1]
```diff
@@ -40,11 +40,12 @@ const useStyles = makeStyles((theme: Theme) => {
     },
     actionButtonWrapper: {
       height: '80px',
-      width: '100%',
+      width: '93%',
```
Comment (taki9) [inline apps/sensenet/src/components/document-viewer.tsx:43]: remove width, and set left: 0

File: apps/sensenet/src/components/editor/sn-monaco-editor.tsx [hunk 2]
```diff
@@ -12,12 +12,12 @@ const useStyles = makeStyles(() => {
   return createStyles({
     actionButtonWrapper: {
       height: '80px',
-      width: '100%',
+      width: '93%',
```
Comment (taki9) [inline apps/sensenet/src/components/editor/sn-monaco-editor.tsx:15]: remove width, and set left: 0

File: apps/sensenet/src/components/view-controls/common/styles.ts [hunk 3]
```diff
@@ -27,11 +27,11 @@ const useStyles = makeStyles(() => {
     },
     actionButtonWrapper: {
       height: '80px',
-      width: '100%',
+      width: '93%',
```
Comment (taki9) [inline apps/sensenet/src/components/view-controls/common/styles.ts:30]: remove width, and set left: 0

File: apps/sensenet/src/components/wopi-page.tsx [hunk 4]
```diff
@@ -17,11 +17,12 @@ const useStyles = makeStyles(() => {
     },
     actionButtonWrapper: {
       height: '80px',
-      width: '100%',
+      width: '93%',
```
Comment (taki9) [inline apps/sensenet/src/components/wopi-page.tsx:20]: remove width, and set left: 0

**Review by taki9 — COMMENTED** [review]
cancel button should not have margin-right when it is alone (no submit button)

**Comment by pusztaienike** [comment]
> cancel button should not have margin-right when it is alone (no submit button)

Done

**Review by taki9 — APPROVED** [review]

**Pull request merged**